packages/webview/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(webview)
+
+if(APPLE)
+	set(WEBVIEW_COMPILE_DEFS "-DWEBVIEW_COCOA=1")
+	set(WEBVIEW_LIBS "-framework WebKit")
+elseif(WIN32)
```

Comment:
this is the part that we need to vet, i think that there is something wrong here. we need a windows expert to look at this

Reply:
creating an issue.